Bottom: a0fd8ec9afbdc4a985daf231cad78f91390ee169
Top:    411076dc3f01abdaea69ac5857900ce5b9d70a98
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:26:25 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ee75899..5817469 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -358,7 +358,7 @@ object3d_from_board_outline (void)
               offset_in_ct = 0;
               ct = ct->next;
               ct_npoints = get_contour_npoints (ct);
-             }
+            }
 
           get_contour_coord_n_in_mm (ct, offset_in_ct, &x1, &y1);
           vertices[i]           = make_vertex3d (x1, y1, -COORD_TO_MM (HACK_BOARD_THICKNESS)); /* Bottom */
